pages/linux/dockerd.md

```diff
@@ -0,0 +1,20 @@
+# dockerd
+
+> Starting docker daemon, persistent process that manages docker containers.
```

Comment:
I feel like this is not really clear. 
Perhaps this will read better? 
```suggestion
> A persistent process to start and manage docker containers.
```

Reply:
I feel like we still need to point out that this command is going to start docker daemon. What about

Starting docker daemon, a persistent process to start and manage docker containers.

Reply:
All of the descriptions mention "Run docker daemon", so I think we can leave it out from the heading. Managing docker containers is good enough to explain what is happening at a broad level.

Reply:
Good point, I'll change it.